packages/suite/src/views/wallet/send/index.tsx

```diff
@@ -91,6 +94,12 @@ const Send = ({
     return (
         <WalletLayout title="Send" account={selectedAccount}>
             <AccountName account={account} message={accountNameMessage} />
+            <AccountSelector
```

Comment:
AccountSelector component should have own container to avoid importing props just to pass them down (like 'goto')
This way `AccountSelector` could be used in any wallet tab